SJRK-297: add sojustrepairit theme

BlueSlug opened the pull request:
Adds the new SJRK-flavoured `sojustrepairit` theme, closely modeled after `cities` and `learningReflections`

After merging, a new branch should be created, from which the site will be hosted. Naming could follow the established convention: `stories-sojustrepairit-production`

For more information, including domain, CI info and configuration settings, please see the [SJRK-297 Jira](https://issues.fluidproject.org/browse/SJRK-297)

BlueSlug committed:
SJRK-297: added new `sjrk` theme files, more to come

BlueSlug committed:
Merge remote-tracking branch 'upstream/stories-floe-dev' into SJRK-297

BlueSlug committed:
SJRK-297: renamed `sjrk` theme to `sojustrepairit`, added more files

BlueSlug committed:
NOJIRA: removed unused UIO colour definitions

BlueSlug committed:
SJRK-297: implementing SJRK theme styling

cindyli commented:
It would be helpful to add a comment for [this line](https://github.com/fluid-project/sjrk-story-telling/blob/stories-floe-dev/sjrk.storyTelling.server.config.json5.example#L13) to list all available themes as a reminder, as developers change the theme value from time to time for testing and development. Thanks.

BlueSlug committed:
SJRK-297: added documentation for new themes and theme config settings

BlueSlug commented:
I have updated the README and added a couple of notes in the example config file to clarify what options are available. I have also run the browser tests in Chrome 77, Firefox 69 and Edge 44 on Windows 10 v.1903 both from a node server and via the `docker-compose` dev build, and everything passed without issue.

Ready for more review

cindyli commented on sjrk.storyTelling.server.config.json5.example:
```diff
@@ -6,16 +6,28 @@
 
-                        // if true, saving stories to the database will be enabled
+                        /*
+                         * if true, saving stories to the database will be enabled
```
It might be more accurate to say "adding and editing stories will be enabled".

BlueSlug replied:
Good catch! I'll make that change

BlueSlug replied:
what do you think of "creating and saving stories..." instead? "Editing" might imply that they can be changed after creation (I realize that the page is called "edit")

cindyli replied:
Ya, the change makes sense.

cindyli commented:
1. Open any SJRK webpage in a narrower browser;
2. Open UIO and set the text size to 2;
3. Scroll to the the bottom of the page, the section of partner logos continue to display horizontally  and the last IDRC logo is out of the view.

<img width="705" alt="Screen Shot 2019-10-02 at 4 05 26 PM" src="https://user-images.githubusercontent.com/572877/66078171-80e18e00-e52f-11e9-86b2-55289f550591.png">

BlueSlug committed:
SJRK-297: tweaking the wording a little...

BlueSlug committed:
NOJIRA: fix footer logo styling bug surfaced by UIO text-size set to 2

BlueSlug commented:
I've adjusted the footer container (three separate rules for three separate themes) to have a maximum width of 100%. The result is that the logos are a little small at the narrowest width before the styling break (50em), but they seem to fit properly at various widths in several browsers.

cindyli commented:
Merged at a870c8afe9609a484f9005778039a0fe73974545

cindyli commented:
The new branch for hosting SJRK-flavoured theme is created at https://github.com/fluid-project/sjrk-story-telling/tree/stories-sojustrepairit-production
